cypress/utils/setup-utils.js

```diff
@@ -0,0 +1,58 @@
+import {
+  accessOrganization,
+  addMockFacility,
+  createOrganization,
+  getOrganizationsByName, getPatientsByFacilityId,
+  markOrganizationAsDeleted, markPatientAsDeleted,
+  verifyPendingOrganization
+} from "./testing-data-utils";
+import { generateUser } from "../support/e2e";
+
+const createOrgName = (specName) => {
+  return `${specName}-org`;
+}
+
+const createFacilityName = (specName) => {
+  return `${specName}-facility`;
+}
+
+const createAndVerifyOrganization = (orgName) => {
+  const adminUser = generateUser();
+  return createOrganization(orgName, adminUser.email)
+    .then((res) => verifyPendingOrganization(res.body.orgExternalId))
```

Comment:
non-blocking question to maybe open up a larger convo: is there any reason here that we're using `.then` rather than async / await? Guessing that we're maybe just inheriting the style from previous cypress specs, and if we are, maybe worth asking if we want to move to using async / await instead since they're a bit more readable. 

If there's another reason why though pls lmk! Slight suspicion that re-writing the specs using the new paradigm would be a bit complicated.

Reply:
This thread I think will answer your question: https://github.com/cypress-io/cypress/issues/1417 Let me know if you need more info!!